File: Assignment_2/Assignment_1.c
#include<stdio.h>

void Display (int iNo)
{
    int iCnt = 0 ;
    while(iCnt < iNo)
    {
        printf(" * ");
        iCnt++;
    }
    printf("\n");
}
int main()
{
    int iValue = 0;

    printf("Enter Number To print * : ");
    scanf("%d",&iValue);

    Display(iValue);
    
    return 0 ;
}

File: Assignment_11/Assignment_java/Assignment_1.java
import java.util.*;

class Pattern 
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        char ch = 'A';
        for(iCnt = 1 ; iCnt <= iNo ; iCnt++,ch++)
        {
            System.out.print(ch+"\t");
        }
        System.out.println();
    }
}
class Assignment_1
{


    public static void main(String A[])
    {   
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}